tests/test_daemon.cpp

```diff
@@ -878,6 +921,10 @@ INSTANTIATE_TEST_SUITE_P(Daemon, LaunchImgSizeSuite,
                          Combine(Values("test_create", "launch"),
                                  Values(std::vector<std::string>{}, std::vector<std::string>{"--disk", "4G"}),
                                  Values("1G", mp::default_disk_size, "10G")));
+INSTANTIATE_TEST_SUITE_P(Daemon, DiskSpaceRequestedExceedsAvailableSuite,
+                         Combine(Values("test_create", "launch"),
+                                 Values(std::vector<std::string>{"--disk", "999999999G"})));
```

Comment:
This is the only value ever used by the test (the vector with the 2 args), so there is no point having it as a parameter.